capture/writer-simple.c

```diff
@@ -865,7 +883,12 @@ void writer_simple_init(char *name)
             LOG("WARNING - Will always use first pcap directory for local index");
         }
 
-        config.maxFileSizeB = MIN(config.maxFileSizeB, 0x07ffffffffL);
```

Comment:
The else isn't needed since the max of maxFileSizeB is 1024^4 or 0x10000000000

The goal of the previous maxFileSizeB was to stop folks from making it too big where variable length encoded ints might start using more space than fixed size. Because of the gzip issue we have to allow it, but I still wonder if we should allow folks to create files so big.

Reply:
ok, that's right.